src/UserDetails/UserDetails.js

```diff
@@ -24,10 +32,43 @@ const UserDetails = ({ username }) => {
   }
 
   return (
-    <div>
-      <h2>{data.name}</h2>
-      <p>{data.bio}</p>
-    </div>
+    <UserDetailsContainer>
+      <UserDetailsContainer>
+        <BioContainer>
+          {data.bio ? (
+            <Text size={sizes.small}>{data.bio}</Text>
+          ) : (
+            <Text size={sizes.small}>No Bio Found</Text>
+          )}
+        </BioContainer>
+        <MiddleContainer>
+          <GridItems>
+            <Text>Repos</Text>
+            <Text size={sizes.medium} fontWeight={fontWeight.bold}>
```

Comment:
Can make a seperate file for different types of text that you can use around the app, which would save a lot of code from being repeated.